### PR TITLE
add scenario button

### PR DESCRIPTION
**Pull request by maxchehab**
Screen Shot 2020-12-17 at 1 19 18 PM

Adds a button to trigger an invalid okta attribute scenario.

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -39,5 +39,11 @@
         </div>
     </fieldset>
   </form>
+  
+  <form action="/portal" method="POST">
+    <input hidden class="hidden-field" id="organization" name="organization" type="text" readonly value="<%= @organization.id %>">
+    <input hidden class="hidden-field" id="organization" name="scenario" type="text" readonly value="okta-invalid-attributes">
```
Comment (rabaut) [inline views/index.erb]: Can we make this `id="scenario"`

Comment (maxchehab) [reply views/index.erb]: done

**Comment by willmanduffy** [comment]
Are we just deploying this to staging?

**Comment by maxchehab** [comment]
@willmanduffy I tested this against production and it worked fine, I'm happy to just ship to staging but confident it will work in production.

**Review by rabaut — APPROVED** [review]

**Pull request merged**